tests/test_tokenization_common.py

```diff
@@ -966,24 +970,48 @@ def test_internal_consistency(self):
 
     @require_tokenizers
     def test_encode_decode_with_spaces(self):
-        tokenizers = self.get_tokenizers(do_lower_case=False)
+        tokenizers = self.get_tokenizers(do_lower_case=False, fast=False)
         for tokenizer in tokenizers:
             with self.subTest(f"{tokenizer.__class__.__name__}"):
                 new_toks = [
-                    AddedToken("[ABC]", normalized=False),
-                    AddedToken("[DEF]", normalized=False),
-                    AddedToken("GHI IHG", normalized=False),
+                    AddedToken(
+                        "[ABC]", normalized=True, lstrip=True, rstrip=True
+                    ),  # these are added tokens, they will be normalized....
+                    AddedToken("[DEF]", normalized=True, lstrip=True, rstrip=True),
+                    AddedToken("GHI IHG", normalized=True, lstrip=True, rstrip=True),
                 ]
                 tokenizer.add_tokens(new_toks)
+                # special tokens should not be normalized...
+                tokenizer.add_tokens([AddedToken("[SAMPLE]", normalized=True)], special_tokens=True)
+                print(tokenizer.added_tokens_encoder)
```

Comment:
4 test cases to be made, special/not special, normalized/ not normalized.
Also need to make sure this is equivalent to fast that always sets (special = not normalized)